functions/dw_server/fn_markerCreation.sqf

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
File header should be nowhere or everywhere :)
Or all public functions etc ...

Reply:
That's some old legacy code. This function will probably be properly removed as soon as I'm sure that no mission is using it. Its only role is to replace two or three easy-to-read lines by a single, barely readable, function call. Not quite useful, but was part of the original DW engine, which is now vastly reworked.

Reply:
Resolution through #18